stable/dex-k8s-authenticator/values.yaml

```diff
@@ -0,0 +1,119 @@
+# Default values for dex-k8s-authenticator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mintel/dex-k8s-authenticator
+  tag: 1.1.0
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+healthCheckPath: ""
+
+dexK8sAuthenticator:
+  # Specify multiple extra root CA files to be loaded
+  # trusted_root_ca:
+  #   -|
+  #     -----BEGIN CERTIFICATE-----
+  #     ...
+  #     -----END CERTIFICATE-----
+
+  # Specify path to tls_cert and tls_key - if enabled, set liten to use https
+  # tls_cert: /path/to/dex-client.crt
+  # tls_key: /path/to/dex-client.key
+
+  # CA for your IDP - used in WebUI instructions for generating
+  # kubeconfig
+  # Both idp_ca_uri and idp_ca_pem are optional - you typically specifiy
+  # one or the other if required
+  #
+  # Provides a link to the CA from a hosted site
+  # idp_ca_uri: http://url-to-your-ca.crt
+  #
+  # Provides abililty to specify CA inline
+  # idp_ca_pem: |
+  #   -----BEGIN CERTIFICATE-----
+  #   ...
+  #   -----END CERTIFICATE-----
+
+
+  # Which address to listen on (set to https if tls configured)
+  listen_port: 5555
+
+  # A path-prefix from which to serve requests and assets
+  web_path_prefix: /
+
+  # Optional kubectl version which provides a download link to the the binary
+  kubectl_version: v1.13.1
+
+  # Optional Url to display a logo image
+  # logo_uri: http://<path-to-your-logo.png>
+
+  # Enable more debug
+  debug: false
+
+  clusters:
+  - name: example-cluster
+    # Descriptions used in the WebUI
+    short_description: "Example Cluster"
+    description: "Example Cluster Long Description..."
+    # Redirect Url pointing to dex-k8s-authenticator callback for this cluster
+    # This should be configured in Dex as part of the staticClients
+    # redirectURIs option
+    redirect_uri: http://127.0.0.1:5555/callback/example-cluster
+    # Client Secret - should match value in Dex
+    client_secret: ZXhhbXBsZS1hcHAtc2VjcmV0
```

Comment:
client secret? Are you sure?

Reply:
ah, that is example config, good then, I would actually replace with:
```
client_secret: CHANGEME
```

Reply:
this is example :)